Sources/BraintreeThreeDSecure/BTThreeDSecureRenderTypes.swift

```diff
@@ -0,0 +1,22 @@
+import Foundation
+
+/// Render types that the device supports for displaying specific challenge user interfaces within the 3D Secure challenge.
+@objcMembers public class BTThreeDSecureRenderType: NSObject {
+
+    public typealias StringValue = String
+
+    /// OTP
+    public static let otp: StringValue = "CardinalSessionRenderTypeOTP"
+
+    /// HTML
+    public static let html: StringValue = "CardinalSessionRenderTypeHTML"
+
+    /// Single select
+    public static let singleSelect: StringValue = "CardinalSessionRenderTypeSingleSelect"
+
+    /// Multi Select
+    public static let multiSelect: StringValue = "CardinalSessionRenderTypeMultiSelect"
+
+    /// OOB
+    public static let oob: StringValue = "CardinalSessionRenderTypeOOB"
```

Comment:
Would the `@objc` annotation work for us ([stack post](https://stackoverflow.com/a/28436181))?

Reply:
That works for the enum but that's not the issue, the issue is that arrays of `Int` enums don't exist in Obj-C and enums can only be bridged to `Int` types to be interoperable with Obj-C. A Swift array is bridged to Obj-C's `NSArray`, and an Obj-C array should contain only pointers, but an enum of `Int` doesn't contain pointers so cannot be bridged.

Reply:
Ah got it yeah. That makes sense. Swift does have `OptionSet` as an `NS_OPTIONS` equivalent, but I can't tell if option sets are accessible from Obj-C.

Reply:
🤔 we may be able to make that work - it'll need to be a class instead of a struct since structs don't exist in Obj-c, but I think that may be okay. I'll poke around at it a bit.

Reply:
Done: https://github.com/braintree/braintree_ios/pull/1134/commits/5911c551bbc829b3002f5f98941f1a248c97f988. Great suggestion, that's way cleaner 🔥

Reply:
Also if anyone is interested this is how the implementation will look for Obj-C merchants:
```objective-c
BTThreeDSecureRequest *request = [[BTThreeDSecureRequest alloc] init];
request.renderTypes = @[BTThreeDSecureRenderType.oob, BTThreeDSecureRenderType.html, BTThreeDSecureRenderType.singleSelect, BTThreeDSecureRenderType.multiSelect, BTThreeDSecureRenderType.otp];
```